### ISSUE TITLE
Teaser Grid Block: clicking on close icon "x" should reset the subblock, not go back to the blocks chooser

### ISSUE THREAD

**Issue by tisto** [plone/volto#5478]
**Describe the bug**

When I click on the close icon "x" of a teaser or image block in a grid, the subblock is completely removed.

**To Reproduce**

1. As admin
2. Create a page with a grid block and a teaser/image
3. Click on the close "x" button on the teaser/image block
4. current behavior: the block is removed

**Expected behavior**

For a client project we amended the functionality to just reset the block (image block without an image, teaser block without a teaser) instead of going back to the block chooser.

At first we were reluctant on that change. Though, working with it for a few weeks revealed that this makes so much more sense than our current Volto core solution.

**Screenshots**

<img width="820" alt="Screenshot 2023-12-02 at 07 48 10" src="https://github.com/plone/volto/assets/632105/8a8588b1-0705-4122-9490-9cf6f2a72a0f">

**Software (please complete the following information):**

Latest Volto / Plone.

**Additional context**

I'd consider this to be a UX bugfix. However, this is up to the release manager or Volto team to decide.

**Comment by tisto** [comment]
@sneridagh I just ran into this and I was shocked when I realized we never pushed this upstream. At first I was reluctant on this change in our client project. Though, now that I ran into the "old" behavior of Volto core I see that this change makes so much sense UX-wise. What do you think?

**Comment by sneridagh** [comment]
It does, it's in the first iteration stage, take a look:

<img width="998" alt="image" src="https://github.com/plone/volto/assets/486927/9a4450b3-a49b-4b28-93ae-d0f0d2f57da6">

top, right, the reset is in there.

It did not "evolved" as in DLR. I think that @albertcasado also weigh in on this one, and had to do something with it. I can't remember the whole thought path that this followed through the years.

**Comment by tisto** [comment]
@sneridagh I know that there is a trash icon in the sidebar. It is just very counterintuitive in comparison to the "close icon" approach. The current solution requires two mouse clicks instead of just one. The functionality is hidden when you actually need it (when looking at the block as a whole, you don't see the trash icon, you have to click inside the particular block to see the trash icon and delete it, this is a UX anti-pattern IMHO).

We can discuss this with Albert and get his opinion/approval.

**Comment by sneridagh** [comment]
I don’t get where is the additional click…

**Comment by tisto** [comment]
@sneridagh:

- go to https://volto.kitconcept.dev/edit
- you can click on the "x" icon of the first grid block and this could reset the block

instead, you have to:

- click on the particular subblock in the grid (e.g. the teaser)
- click on the trash icon to delete it

Two mouse clicks instead of one. Hiding the functionality from the user in the 90% use case (that you don't have the subblock selected that you want to delete).

**Comment by sneridagh** [comment]
Ok, got it. 

Indeed, the evolution that this had in both light theme and in the client project didn't get into core. The core version was mainly developed in sprints, mainly in Innsbruck beginning of this year. Then the others evolved as the core one stayed with the original implementation.

**Comment by sneridagh** [comment]
Uh, I just realised that the light theme has the core version.

**Comment by tisto** [comment]
@sneridagh ok, we need to make a decision where to go then. 

I started to work with the Volto Light Theme daily for our new website. I am ok with reporting every single issue I find. However, it is important for me that we find a way to either push those things upstream or find another way that we don't have to apply those small improvements over and over again.

I am ok to put this into the Light Theme, if that's the right place in your opinion. I am just the product owner guy here :)

**Comment by akshayw1** [comment]
I would like to work on this!Please assign

**Comment by stevepiercy** [comment]
@akshayw1 please read and follow [Contributing to Plone](https://6.docs.plone.org/contributing/index.html) and [First-time contributors](https://6.docs.plone.org/contributing/first-time.html).

**Comment by kanhaiya04** [comment]
@tisto, I comprehend the modifications you've outlined regarding the 'x' icon. If your decision is to alter the functionality of the 'x' icon, I am eager to undertake this task.

**Comment by stevepiercy** [comment]
@kanhaiya04 this issue needs further direction from @plone/volto-team before any work is performed. It's not clear what @sneridagh @tisto and the rest of the team decided to do for this issue.

**Comment by stevepiercy** [comment]
@draunger GitHub does not use `/assign`. Stop spamming. Thank you.

**Comment by draunger** [comment]
@stevepiercy I want to work on this issue how I assign me??

**Comment by stevepiercy** [comment]
No. Read the [previous comment](https://github.com/plone/volto/issues/5478#issuecomment-1870775528).

**Comment by sneridagh** [comment]
@tisto did we decided to hold up on this one, until we have a better UX for this, right? Since we realised that the implemented version is the most "correct" one for now. I'd close this one until we don't have anything better.
